RELEASE.md

```diff
@@ -29,6 +29,7 @@
   * Converted `setup.py` in new Kedro project template and Kedro starters to `pyproject.toml` and moved flake8 configuration
   to dedicated file `.flake8`.
   * Updated the spaceflights starter to use the new flat `conf/` structure.
+* Allowed using of custom cookiecutter templates for creating pipelines with `--template` flag for `kedro pipeline create` or via `template/pipeline` folder.
```

Comment:
@jasonmhite Would you move this to be under the "Upcoming release 0.18.14" heading since we already release 0.18.13? :)

Reply:
On it, I was just merging and finally getting some time to expand the docs.